### ISSUE TITLE
Swift 5

### ISSUE THREAD

**Issue by sebastienboulogne** [Awalz/SwiftyCam#209]
Hello guys, thank you for your work. Is it possible to compile your pod for Swift 5 because i'm blocked with the new Xcode :/ Thank you very much

**Comment by mtrezza** [comment]
I compiled it with carthage using the Swift 5.0.1 compiler, if that helps.

**Comment by sebastienboulogne** [comment]
@mtrezza thank you but using Pod. Waiting for the Swift 5 version

**Comment by ShaikBaji1911** [comment]
I have one query if we are using any other camera (video call in Skype or Fb,Whatssup) but can we use the SwiftCam Camera in parallel there I used to see the Blank screen how to detect and make a popup there like Camera in use already

**Comment by bkunarola** [comment]
is there any update on this?

**Comment by mtrezza** [comment]
@ShaikBaji1911 Please open a separate issue for this, if it doesn’t exist yet.

**Comment by ShaikBaji1506** [comment]
Does this work on Swift 5 Xcode 11 ? when we update Xcode I was adding though cocoa pods

**Comment by caesss** [comment]
any news?

**Comment by AbdulRahman-Khursheed** [comment]
Waiting for swift 5 version

**Comment by ryanstewart1996** [comment]
> Waiting for swift 5 version

same here, fingers crossed
